pdm.lock

```diff
@@ -73,6 +73,48 @@ files = [
     {file = "Babel-2.12.1.tar.gz", hash = "sha256:cc2d99999cd01d44420ae725a21c9e3711b3aadc7976d6147f622d8581963455"},
 ]
 
+[[package]]
```

Comment:
Black is being readded here. 🤔

Reply:
Or is it intended?

Reply:
Hmm didn't look at the diff haha. @sydney-runkle you might want to delete `pdm.lock` and rebuild it from scratch

Reply:
I think it has to be added because `pytest-examples` still uses it, but I can switch that to `ruff` today

Reply:
I can make the `pytest-examples` changes before we merge this so that `black` isn't added

Reply:
Actually, going to go ahead and merge this. I do think we should change `pytest-examples`, but ripping out `black` changes the API pretty significantly, and I think it might merit a V2. I'll tackle these changes, but I think we should go ahead and update the `pdm.lock` file in the meantime.